docs-2.0/3.ngql-guide/14.native-index-statements/1.create-native-index.md

```diff
@@ -1,8 +1,18 @@
+!!! warning 
+
+    索引的概念和使用限制都较为复杂。索引配合 `LOOKUP` 和 `MATCH` 语句使用。
+
 # CREATE INDEX
 
-`CREATE INDEX`语句可以为现存的标签、边类型或属性创建原生索引。
+`CREATE INDEX`语句用于对Tag、EdgeType或其属性创建`原生索引`。通常分别称为`标签索引`、`边类型索引`、`属性索引`。
+
+!!! note
 
-大多数图查询都是从属性标识的点或边的列表开始检索的。索引可以让大规模图数据库的全局检索更加高效。
+    `标签索引`和`边类型索引`用于`和标签、边类型（自身）相关`的查询,而不是`基于标签上某属性`的查询。例如用 LOOKUP 找到`某种边类型E`的所有的边。
+    
+    `属性索引`目的是`基于属性的查询`。例如基于属性`Age`找到`Age == 19`的所有的点VID。
+    
+     如果已经为`某个标签T的属性A`建立过属性索引 `i_TA`，则没有必要对于`标签T`额外再建立一个标签索引`i_T`。这是因为查询引擎可以使用`i_TA`来替代`i_T`。边类型索引同理。但是，`i_T`却不能替代`i_TA`用于属性查找。
```

Comment:
No need to use note here. These lines are normal instructions.

Reply:
ok